packages/desktop-client/src/components/schedules/SchedulesTable.js

```diff
@@ -93,7 +93,7 @@ export function ScheduleAmountCell({ amount, op }) {
         <View
           style={{
             textAlign: 'left',
-            color: colors.n7,
+            color: theme.tableText,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏wrong color

before
<img width="75" alt="Screenshot 2023-08-12 at 09 33 03" src="https://github.com/actualbudget/actual/assets/886567/c936edd8-f4e3-4895-98d7-ad4ac71fecc4">


after
<img width="98" alt="Screenshot 2023-08-12 at 09 33 01" src="https://github.com/actualbudget/actual/assets/886567/c406fb8f-9381-426d-a1e8-ebc07afb5843">